.github/workflows/python-publish.yml

```diff
@@ -67,3 +67,44 @@ jobs:
         DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
       with:
         args: 'Launchable CLI ${{ needs.tagpr.outputs.tag }} is released! https://github.com/launchableinc/cli/releases/tag/${{ needs.tagpr.outputs.tag }}'
+
+  docker:
+    name: Push Docker image to Docker Hub
+    needs: tagpr
+    if: needs.tagpr.outputs.tag != '' || github.event_name == 'workflow_dispatch'
+
+    runs-on: ubuntu-22.04
+
+    permissions:
+      packages: write
+      contents: read
+      attestations: write
+      id-token: write
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push Docker image
+        id: push
+        uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # v6.18.0
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms:
+            - linux/amd64
+            - linux/arm64
+          push: true
+          tags: cloudbees/launchable:${{ needs.tagpr.outputs.tag }}
+
+      - name: Generate artifact attestation
+        uses: actions/attest-build-provenance@db473fddc028af60658334401dc6fa3ffd8669fd # v2.3.0
+        with:
+          subject-name: index.docker.io/my-docker-hub-namespace/my-docker-hub-repository
```

Comment:
Is it correct?

Reply:
Oh, it's not correct. Thank you for pointing it out.